app/routes/pages/components/Sidebar.js

```diff
@@ -51,9 +51,10 @@ class Sidebar extends Component<Props, State> {
                 <Icon name="close" size={50} />
               </button>
               <h3 className={styles.sidebarHeader}>Om Abakus</h3>
+              {/*
               <h4 className={styles.sidebarSubtitle}>
                 {categorySelected ? categorySelected : 'Generelt'}
-              </h4>
+              </h4>*/}
```

Comment:
Do you need this comment here ?

Reply:
I'd say you can just remove it entirely.